editor/scene_tree_dock.cpp

```diff
@@ -2110,6 +2176,13 @@ SceneTreeDock::SceneTreeDock(EditorNode *p_editor, Node *p_scene_root, EditorSel
 	filter_hbc->add_child(tb);
 	button_instance = tb;
 
+	tb = memnew(ToolButton);
+	tb->connect("pressed", this, "_tool_selected", make_binds(TOOL_EXPAND_COLLAPSE, false));
+	tb->set_tooltip(TTR("Expand/Collapse all nodes for the selected node"));
```

Comment:
Could you add a period at the end of the phrase?

Reply:
Not only that, fixed code typos as well, thanks.
Some strings are missing periods too, another pr is worth to fix them all!